sdk/eventhub/testhub/package.json

```diff
@@ -36,7 +36,7 @@
     "@azure/event-hubs": "^2.1.4",
     "@azure/event-processor-host": "^2.0.0",
     "@types/node": "^8.0.0",
-    "@types/uuid": "^3.4.3",
+    "@types/uuid": "^8.0.0",
```

Comment:
@ramya-rao-a, @chradek: Same question about updating `event-processor-host` applies to `eventhub/testhub`.  Update for consistency with other packages, or keep using old version?

Reply:
I'm fine with updating to stay consistent.